src/domain/occurrence/__tests__/CreateOccurrencePage.test.tsx

```diff
@@ -1232,6 +1233,148 @@ describe('enrolment type selector', () => {
   );
 });
 
+describe('auto acceptance for enrolments', () => {
+  it('renders auto acceptance message when auto accept is set on', async () => {
+    renderComponent({
+      mocks: [getEventMockedResponse({ autoAcceptance: false })],
+    });
+    // Wait for form to have been initialized
+    await screen.findByTestId('time-and-location-form');
+    // set as checked
+    act(() => userEvent.click(getFormElement('autoAcceptance')));
+    await waitFor(() => {
+      expect(getFormElement('autoAcceptanceMessage')).toBeInTheDocument();
+    });
+    expect(
+      screen.getByRole('heading', {
+        name: /vahvistusviesti sisältää automaattisesti seuraavat tiedot/i,
+      })
+    ).toBeInTheDocument();
+    expect(
+      screen.getByText(
+        /personoitu tervehdys, ilmoittautuminen vahvistettu, tapahtuman tiedot, aika, varattujen paikkojen lukumäärä, kieli, paikka, osoite, järjestäjän yhteystiedot\./i
+      )
+    ).toBeInTheDocument();
+  });
+
+  it('submits the auto acceptance message right', async () => {
+    const Utils = require('../utils');
+    const spyGetEditEventPayload = jest.spyOn(Utils, 'getEditEventPayload');
+    const toastSuccess = jest.spyOn(toast, 'success');
+    const enrolmentStart = '2021-05-03T21:00:00.000Z';
+    const enrolmentEndDays = 1;
+    const neededOccurrences = 1;
+    renderComponent({
+      mocks: [
+        getEventMockedResponse({
+          location: true,
+          autoAcceptance: false,
+          enrolmentEndDays,
+          enrolmentStart,
+          neededOccurrences,
+        }),
+        getUpdateEventMockResponse({
+          autoAcceptance: true,
+          autoAcceptanceMessage: 'bla bla',
+          enrolmentEndDays,
+          enrolmentStart,
+          neededOccurrences,
+        }),
+      ],
+    });
+    // Wait for form to have been initialized
+    await screen.findByTestId('time-and-location-form');
+    // set as checked
+    act(() => userEvent.click(getFormElement('autoAcceptance')));
+    await waitFor(() => {
+      expect(getFormElement('autoAcceptanceMessage')).toBeInTheDocument();
+    });
+    userEvent.type(getFormElement('autoAcceptanceMessage'), 'bla bla');
```

Comment:
Wondering if the message you are using here is a kind of PoC message and whether you intended to align it with he usecase later? Up to you.

Reply:
I'm not sure if this part was here when you reviewed, but the thing is that I would like to test whether or not the value really changes in formik context and what is really posted to API.
```
    expect(spyGetEditEventPayload).toHaveBeenCalledWith({
      event: expect.anything(),
      formValues: expect.objectContaining({
        autoAcceptance: true,
        autoAcceptanceMessage: 'bla bla',
      }),
    });
    expect(spyGetEditEventPayload).toHaveReturnedWith(
      expect.objectContaining({
        pEvent: expect.objectContaining({
          translations: [
            { languageCode: 'FI', autoAcceptanceMessage: 'bla bla' },
          ],
        }),
      })
    );
```

Reply:
the auto acceptance message is the whole beef here, so I'm not sure if I understood your comment properly :) Api changes are here (like mentioned in the start comment) https://github.com/City-of-Helsinki/palvelutarjotin/pull/243